utils/service_discovery/sd_docker_backend.py

```diff
@@ -89,8 +89,11 @@ def __init__(self, agentConfig):
 
         self.docker_client = DockerUtil(config_store=self.config_store).client
         if Platform.is_k8s():
-            self.kubeutil = KubeUtil()
```

Comment:
nit: should we assign `self.kubeutil = None` in case the exception is raised, to avoid attribute errors later?